indico/htdocs/js/indico/modules/categories/management.js

```diff
@@ -301,4 +267,111 @@
             }
         }).isEverythingSelected;
     };
+
+    function _fetchSourceCategory(categoryId) {
+        if (!_categories[categoryId]) {
+            _categories[categoryId] = categoryId;
+            $.ajax({
+                url: build_url(Indico.Urls.Categories.info, {category_id: categoryId}),
+                dataType: 'json',
+                error: handleAjaxError,
+                success: function(data) {
+                    if (data) {
+                        _categories[categoryId] = data;
+                    }
+                }
+            });
+        }
+    }
+
+    function _moveCategories(ids, source, endpoint) {
+        var sourceId = _.isObject(source) ? source.category.id : source;
+        var data = {category_id: ids};
+
+        $('<div>').categorynavigator({
+            category: source,
+            confirmation: true,
+            openInDialog: true,
+            actionButtonText: $T.gettext("Move here"),
+            dialogTitle: $T.ngettext("Move category", "Move categories", ids.length),
+            dialogSubtitle: $T.ngettext("Select new category parent for the category",
+                                        "Select new category parent for {0} selected categories",
+                                         ids.length).format(ids.length),
+            actionOn: {
+                categoriesWithEvents: {disabled: true},
+                categoriesDescendingFrom: {
+                    disabled: true,
+                    ids: data.category_id
+                },
+                categories: {
+                    disabled: true,
+                    groups: [{
+                        ids: [sourceId],
+                        message: $T.ngettext("The category is already here",
+                                             "Selected categories are already here",
```

Comment:
Not always. This is also used when moving the current category or clicking on the move icon for a subcategory.